### ISSUE TITLE
Assets are copied incorrectly

### ISSUE THREAD

**Issue by petkaantonov** [a7ul/esbuild-node-tsc#20]
Assets are not copied in the way `tsc`copies them. It simply copies assets relative to the outDir root, not where they were found and then runtime requires fail.

For example (outdir=lib,srcDir=src:

- project/src/code.ts
- project/src/asset.json

results in:

- project/lib/asset.json
- project/lib/project/code.ts

Correct result (and what tsc gives):

- project/lib/project/asset.json
- project/lib/project/code.ts

**Comment by eeston** [comment]
I have the same issue.

**Comment by a7ul** [comment]
Just made the outDir configurable for assets.
So you could now do this. Hope that helps. 
This is available in v1.6.0

```
{
  outDir: "./lib",
  assets: {
    baseDir: "src",
    outDir: "./lib/project",
    filePatterns: ["**/*.json"],
  }
}
```
